src/main/java/ch/njol/skript/conditions/CondIsAdult.java

```diff
@@ -0,0 +1,61 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import org.bukkit.entity.Ageable;
+import org.bukkit.entity.Animals;
+import org.bukkit.entity.LivingEntity;
+
+@Name("Is An Adult")
+@Description("Tells whether or not a living entity is an adult.")
+@Examples({
+	"on spawn:",
+	"\tif event-entity is not an adult:",
+	"\t\tmake event-entity an adult"
+})
+@Since("INSERT VERSION")
+public class CondIsAdult extends PropertyCondition<LivingEntity> {
+
+
+	static boolean HAS_MOB_SUPPORT = Skript.isRunningMinecraft(1,16,5);
```

Comment:
Where is this documented?

Reply:
This isn't actually documented well and I should probably add a note if any future edit is needed. 
When I first made this my test was using zombie and it worked fine up until I reached Java8 at which point kept failing but all the code was correct. I spent forever on the docs and the only change was between 1.15.2 & 1.16.5

**Pre 1.16.5**
![image](https://user-images.githubusercontent.com/81952476/230692617-b9054cdb-5f56-49af-948f-e5a70a544031.png)

**Post 1.16.5**
![image](https://user-images.githubusercontent.com/81952476/230692642-a9482fbf-32bf-4db2-a705-76126c0a33b8.png)

With the fact the method had no changes I could really check using `Skript#isRunningMinecraft` was my only option

Reply:
I think a comment left in code is good, do you think we should add a note for users too?

Reply:
As long as required plugins is readable it should suffice, but if you believe so I can see about adding a more noticeable note in description